### ISSUE TITLE
Blockquote Component

### ISSUE THREAD

**Issue by alexfinnarn** [CUCentralAdvancement/essential-cu#323]
The blockquote component is only used on the Story page currently. I think the design is okay on desktop view but not for mobile screens...at least that was reported to me.

This issue will look at the blockquote and work towards a design resolution for all screen sizes.

**Design Changes**

- [ ] [Blockquote in desktop](#bq-desktop) - This might look okay but the max width should be determined.
- [ ] [[Blockquote in mobile](#bq-mobile) - The padding and larger font make the quote very long.
- [ ] [[Blockquote design idea](#bq-idea) - What if it were down to the icon padding and italics?

**Explanations**

<span id="bq-desktop"></span>
### Blockquote in desktop

<img width="1102" alt="Screen Shot 2021-10-08 at 4 43 19 PM" src="https://user-images.githubusercontent.com/3640707/136628018-d9bf25c8-06f0-4405-a4c8-d05fcf2352bb.png">

I kept other components in the screenshot to show max widths between them. Before correcting anything else in the blockquote design, the width is important since the quotemark icon is placed over the top border and requires absolute values. 

- Blockquote - `max-width: 479px` - screen small
- TextBlock - `max-width: 719px` - screen medium
- Image - `max-width: 1023px` - screen large

I'm setting components max-width based on the proportion I see in the design and then what that most closely resembles in the most used breakpoints for the IR20 project last year.

What should the max-width be for blockquote in screen > 1023px size?

<span id="bq-mobile"></span>
### Blockquote in mobile

<img width="241" alt="Screen Shot 2021-10-08 at 5 11 42 PM" src="https://user-images.githubusercontent.com/3640707/136630111-2df203f7-9380-476c-beaa-83e2abf6b145.png">

I know the look on mobile was discussed before and desire to change it. I'm not really sure what else to add here...

<span id="bq-idea"></span>
### Blockquote design idea

<img width="251" alt="Screen Shot 2021-10-08 at 5 21 53 PM" src="https://user-images.githubusercontent.com/3640707/136630779-6b54c850-c7cd-43ba-ba16-7abc4f6262bb.png">

Since the padding is an issue, it would be best to only have it on the left side. The borders also cause issues with alignment. I have seen a left border used on a news site's blockquotes recently, but I forget where that was. 

By default, the browser adds padding and italics to blockquote, so I think those two elements signify to users the text is a quote. The only other element I would add is the quotation mark. 

I did think that the background color could change and signify something, but then on the desktop that would probably look weird.

**Comment by alexfinnarn** [comment]
@duquennoisa Let me know if this format works. I have similar questions on what I call the "Related Funds Disclosure" but I want to add those in a way that works.

**Comment by alexfinnarn** [comment]
@duquennoisa Yes, this does help. I'm not sure how helpful the issue format is for discussing stuff like this so I will go ahead and try the new Slack video tool and post in the IR21 channel with my update. Then, if that works, I'll close this issue.

**Comment by alexfinnarn** [comment]
Nope...my video was garbage so I didn't post it. I did update the staging site, though. For easy reference: https://essential-staging-cu.herokuapp.com/impact-reports/joy/stories/story-one

I centered the text since that's how the NYT style appeared, and I think it will make sense once real quotes are in with varying character lengths.

I'm not sure how much you use browser dev tools, but feel free to call out measurements from inspecting things that way. For instance, the left/right margins on the quote are `1.5rem` which matches the padding on the paragraph above the quote...I used margins on the blockquote instead of padding due to the border, and I suppose both elements could use margin...but I digress.

I've updated font sizes and line heights to what InVision had, but paddings/margins still need to be worked out, which we can talk about. 

For this issue though, let me know any feedback on the blockquote that I updated.

**Comment by duquennoisa** [comment]
@alexfinnarn the block quote looks good! 
Lets make it full width on desktop too. More like this: 
![image](https://user-images.githubusercontent.com/70037329/136950823-eec5d115-656b-4ec1-8bc4-3e2af52bb0cd.png)

I thinking now it will work better with the image blocks to keep the text consistent width.
